tensorflow/lite/micro/testing/micro_test.h

```diff
@@ -153,6 +153,23 @@ extern tflite::ErrorReporter* reporter;
     }                                                                         \
   } while (false)
 
+#define TF_LITE_MICRO_EXPECT_NEAR_COUNT(x, y, count, epsilon, max_errs)      \
```

Comment:
Great to see this, thanks, it will be useful! Could you add errs_count to the micro_test namespace, like the other variables?